dart/lib/src/hub.dart

```diff
@@ -32,20 +35,18 @@ class Hub {
     return Hub._(options);
   }
 
-  Hub._(SentryOptions options)
-      : _options = options,
-        _stack = ListQueue() {
+  Hub._(SentryOptions options) : _options = options {
     _stack.add(_StackItem(_getClient(fromOptions: options), Scope(_options)));
     _isEnabled = true;
   }
 
   static void _validateOptions(SentryOptions options) {
     if (options == null) {
-      throw ArgumentError.notNull('options');
+      throw ArgumentError.notNull('SentryOptions is required.');
     }
 
-    if (options.dsn == null) {
-      throw ArgumentError.notNull('options.dsn');
+    if (options.dsn == null || options.dsn.isEmpty) {
```

Comment:
is there a nicer way to test null and empty?

Reply:
```dart
if (options.dsn?.isNotEmpty != true ) { //...}
```
is it nicer ?